### ISSUE TITLE
Incorrect TRRS schematic symbol resulting in incorrect PCB footprint

### ISSUE THREAD

**Issue by shadyan** [keebio/bfo9000-pcb#1]
# Incorrect TRRS schematic symbol resulting in incorrect PCB footprint

The pins on the TRRS jack schematic symbol are incorrectly numbered, which results in incorrect wiring in the PCB. See incorrect writing PCB image below:

![image](https://user-images.githubusercontent.com/11218993/217865782-7f3f1b66-d20b-4d35-aa76-432885e3d8ce.png)

I was able to confirm that the wiring is indeed incorrect on a physical BFO-9000 PCB using a multimeter.

See below a screenshot of incorrect and correct schematics:
![image](https://user-images.githubusercontent.com/11218993/217863163-8248b157-8b3d-4438-b117-e30ad795d71a.png)

## Extent of the effect

I looked into other places where this issue could come up.

I tested the symbols in the following Keebio Components library and the pins were correct, so that's good:
[https://github.com/keebio/keebio-components](https://github.com/keebio/keebio-components)

I did however, find the same issue in the following places:

- [Quefrency Rev. 1 Keyboard PCB](https://github.com/keebio/quefrency-rev1-pcb) 
- [Various schematics PDF files in Keebio Docs](https://github.com/keebio/keebio-docs/tree/master/static/schematics)

## Suggested fix

Suggested steps for fix:
1. Update the symbols in the affected schematics to match what's on the PCBs.
2. Version the PCBs (e.g. BFO-9000 v1.0) so that it's possible to identify which type of board you have if the PCBs end up changing.
3. Update the symbols so that in the resulting PCB the VCC line is connected to the Tip of the TRRS jack and bump the PCB version (e.g. BFO-9000 v1.1 or v2.0).

I think just updating the schematics would be great, and I completely understand that people should not be pulling out or plugging in the TRRS cables when the keyboard's main MCU is connected.

> **Note**
> I wrote this up so that others don't have to go through the same troubleshooting steps that I did and hopefully this Github Issue will be of use for those in the same boat as me, until the affected places get updated.

**Comment by nooges** [comment]
Yeah, the original symbol for the TRRS used for the BFO-9000 and a couple of other boards was incorrect, and later boards used the corrected symbol. These files are being provided as-is without any further modifications on my end, so if someone else wants to fix this issue, they are free to do so and can submit a PR for it.
